docs/src/content/docs/guides/customization.mdx

```diff
@@ -139,7 +141,7 @@ tableOfContents:
   </TabItem>
   <TabItem label="Global config">
 
-```js
+```js {7}
 // astro.config.mjs
 
 defineConfig({
```

Comment:
Not strictly speaking related to this PR, but noticing again the weird empty title in this example, maybe an opportunity to fix it?

Something like this?

```js
			title: 'Docs with custom table of contents config',
```